client/components/mma/upgrade/UpgradeSupportSwitchThankYou.tsx

```diff
@@ -54,7 +54,7 @@ export const UpgradeSupportSwitchThankYou = () => {
 						<SvgEnvelope size="medium" />
 						<span data-qm-masking="blocklist">
 							You will receive a confirmation email to{' '}
-							<> {userEmailAddress} </>
+							<> {userEmail} </>
```

Comment:
the JSX fragment seems unnecessary here, in fact it looks like the there isn't a space in the screenshot?
 
![image](https://github.com/guardian/manage-frontend/assets/114918544/36287c1d-7eb5-433f-9b07-f91459de2d51)

Reply:
This was a hangover from previous way of passing in the email address